### DISCUSSION TITLE
Elementary for ARM devices

### DISCUSSION THREAD

**Discussion by pauloricardosb** [elementary/triage#590] [Q&A] [answered]
Do you have any plans or timeline to release an ARM-compatible version of the Elementary OS, considering the growing market presence and technological advancements of ARM processors?

**Comment by cyrneko** [comment, marked as answer]
Technically speaking, albeit not publicized, there are already two ARM builds that are built daily: Raspberry Pi (4) and Pinebook Pro.

Well, the Pinebook Pro builds all fail but you get the deal.

I'd wager a guess and say this means they are definitely interested :)

**Comment by ylluminate** [comment]
Any news on updates for this? Would like to run via Parallels on macOS as a test and perhaps to do some dev work with it.

**Reply by cyrneko** [reply]
Not from danielle's (or anyone else on the team) side to my knowledge. Time and funding is quite limited so I would wager a guess and say this probably isn't happening soon unless someone steps up and puts in the time themselves. *In theory* everything should build for an arm64 target to my knowledge

**Comment by cyrneko** [comment]
@pauloricardosb @ylluminate 

There is updates on this!

There are now ARM64 EFI images available in early access at https://builds.elementary.io

Y'all can get access to that by [sponsoring Elementary](https://github.com/sponsors/elementary) at 1 USD or more
